WeightProducer/python/MCSampleValues.py

```diff
@@ -251,669 +247,501 @@ class MCSampleValuesHelper():
         },
         "TTTo2L2Nu" : {
             "CrossSection" : XSValues(
-                XS_13TeV=88.29, XSSource_13TeV="XSDB (NNLO)",
+                XS_13TeV=687.1, XSSource_13TeV="XSDB (NLO)",
```

Comment:
This is a big change and a step down from NNLO. Where did `687.1` come from? Does it take into account the BR to 2L2Nu?

Reply:
Perhaps, if this is what's missing, we should use the BranchingRatio mechanism (i.e. https://github.com/TreeMaker/TreeMaker/pull/573/files#diff-8fd74e435000e63630ca4a7ff70eb8557d93718085f7e34fcf71f5001b275b99L1032) to make it more clear where the values are coming from.

Reply:
Again, the XSDB delivers no insight into where the 687.1. I see what happened with the Br part as well. Will amend with latest PDG numbers.